test/util/bspNet/docker.ts

```diff
@@ -134,3 +135,13 @@ export const addBspContainer = async (options?: {
 
   return { containerName, rpcPort, p2pPort, peerId };
 };
+
+export const pauseBspContainer = async (containerName: string) => {
+  const docker = new Docker();
+  const container = docker.getContainer(containerName);
+  await container.pause();
+};
+
+export const resumeBspContainer = async (_containerName: string) => {
+  throw new Error("This function is not implemented yet.");
```

Comment:
I believe I am missing something here, isn't this just a call to `unpause`?

Reply:
my first thought would be that the node doesn't know it needs to catch up to head and so gets sync issues. 

My first attempt to impl this will be to mount the kvp db in a temp volume, and restart the container and purge unfinalized blocks. This is similar what devops would do (turn it off and on again).